pkg/tracemeta/tracemeta.go

```diff
@@ -11,4 +11,5 @@ type TraceMeta struct {
 	TimeCreation string `json:"timecreation,omitempty"`
 	TimeDeletion string `json:"timedeletion,omitempty"`
 	Capabilities uint64 `json:"capabilities,omitempty"`
+	Node         string
```

Comment:
Out of curiosity, would it be possible / better to parse it directly from the json to avoid adding extra logic here?

Reply:
We would need traceloop to add it in the annotation. It should be possible. The `traceloop` process already has the `$TRACELOOP_NODE_NAME` environment variable.

Reply:
https://github.com/kinvolk/traceloop/pull/39

Reply:
I opened https://github.com/kinvolk/inspektor-gadget/pull/77, hopefully we can just update the traceloop version without having to update this structure definition.

Reply:
#77 has been merged, I rebased to use the package from traceloop.